src/components/admin/event/AdminPickupEvent/style.module.scss

```diff
@@ -32,6 +32,12 @@
 .submitButtons {
```

Comment:
You should add `flex-wrap: wrap;` here for mobile

![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/55248c2d-f4c8-4466-8c39-db6dd732b031)